googletest/CMakeLists.txt

```diff
@@ -81,6 +81,19 @@ cxx_library(gtest "${cxx_strict}" src/gtest-all.cc)
 cxx_library(gtest_main "${cxx_strict}" src/gtest_main.cc)
 target_link_libraries(gtest_main gtest)
 
+# Targets generated by pump.py.
+pump_target(
```

Comment:
Is there a way to specify these as a list instead of so many pump_target rules?
Also, can we have gtest declare its deps on these instead of having the dep injected "from below"?